controllers/apply/form-router-next/steps.js

```diff
@@ -218,13 +220,29 @@ module.exports = function(formId, formBuilder) {
                     if (preparedFiles.filesToUpload.length > 0) {
                         try {
                             await Promise.all(
-                                preparedFiles.filesToUpload.map(file =>
-                                    uploadFile({
-                                        formId: formId,
-                                        applicationId: currentlyEditingId,
-                                        fileMetadata: file
-                                    })
-                                )
+                                preparedFiles.filesToUpload.map(async file => {
+                                    if (config.get('features.enableLocalAntivirus') && !isTestServer) {
+                                        const avStatus = await checkAntiVirusClamD(formId, currentlyEditingId, file);
+
+                                        if(avStatus.Value === 'CLEAN') {
+                                            uploadFile({
+                                                formId: formId,
+                                                applicationId: currentlyEditingId,
+                                                fileMetadata: file
+                                            });
+                                        } else {
+                                            logger.error('File upload skipped', {
```

Comment:
We'll want to do more than just a `logger.error` here. Need to give this a little thought as to how we shout loudly if we find a virus.

Reply:
Sure ... I followed the same behavior as the existing AV function on finding infected file.

Reply:
Thinking about this I think we are fine here we can used cloudwatch or datadog alerts to give the level of alerting we need for this.